src/XperienceCommunity.Localization.Base/Localizer/XperienceStringLocalizerBase.cs

```diff
@@ -0,0 +1,182 @@
+﻿using AngleSharp.Common;
```

Comment:
Can you remove this unused directive?

Reply:
This one indeed wasn't needed (not sure why it didn't show as greyed out on mine), the rest are needed and produce errors if removed.